cmd/export.go

```diff
@@ -0,0 +1,266 @@
+			pkg.Sequences = make(map[string]parsers.Sequence)
+		}
+
+		pkg.Sequences[wskAction.Name] = *seq
+	} else {
+		parsedAction := *maniyaml.ComposeParsersAction(*wskAction)
+
+		// get the action file extension according to action kind (e.g. js for nodejs)
+		ext := utils.FileRuntimeExtensionsMap[wskAction.Exec.Kind]
+
+		manifestDir := filepath.Dir(utils.Flags.ManifestPath)
+
+		// store function file under action package name subdirectory in the specified manifest folder
+		functionDir := filepath.Join(manifestDir, packageName)
+		os.MkdirAll(functionDir, os.ModePerm)
+
+		// store function in manifest under path relative to manifest root
+		functionFile := filepath.Join(packageName, wskAction.Name) + "." + ext
+		parsedAction.Function = functionFile
+
+		// create function file at the full path
+		functionFile = filepath.Join(manifestDir, functionFile)
+		f, err := os.Create(functionFile)
+		if err != nil {
+			return wskderrors.NewFileReadError(functionFile, err.Error())
+		}
+
+		defer f.Close()
+
+		// store action function in the filesystem next to the manifest.yml
+		// TODO: consider to name files by namespace + action to make function file names uniqueue
+		f.Write([]byte(*wskAction.Exec.Code))
+		pkg.Actions[wskAction.Name] = parsedAction
+	}
+
+	maniyaml.Packages[packageName] = pkg
+	return nil
+}
+
+func ExportCmdImp(cmd *cobra.Command, args []string) error {
+
+	projectName := utils.Flags.ProjectPath
+	maniyaml := &parsers.YAML{}
+	maniyaml.Project.Name = projectName
+
+	config, _ = deployers.NewWhiskConfig(wskpropsPath, utils.Flags.DeploymentPath, utils.Flags.ManifestPath, false)
+	client, _ = deployers.CreateNewClient(config)
+
+	// Init supported runtimes and action files extensions maps
+	setSupportedRuntimes(config.Host)
+
+	// Get the list of packages in your namespace
+	packages, _, err := client.Packages.List(&whisk.PackageListOptions{})
+	if err != nil {
+		return err
+	}
+
+	// iterate over each package to find managed annotations
+	// check if "managed" annotation is attached to a package
+	// add to export when managed project name matches with the
+	// specified project name
+	for _, pkg := range packages {
+		if a := pkg.Annotations.GetValue(utils.MANAGED); a != nil {
+			// decode the JSON blob and retrieve __OW_PROJECT_NAME
+			pa := a.(map[string]interface{})
+
+			// we have found a package which is part of the current project
+			if pa[utils.OW_PROJECT_NAME] == projectName {
+
+				if maniyaml.Packages == nil {
+					maniyaml.Packages = make(map[string]parsers.Package)
+				}
+
+				maniyaml.Packages[pkg.Name] = *maniyaml.ComposeParsersPackage(pkg)
+				// TODO: throw if there more than single package managed by project
+				// currently will be a mess because triggers and rules managed under packages
+				// instead of the top level (similar to OW model)
+				if len(maniyaml.Packages) > 1 {
+					return errors.New("currently can't work with more than one package managed by one project")
```

Comment:
we generally define errors under `wskderrors` with a translatable strings in `en_US.all.json` and `i18n_ids.go`, please open an issue if this PR is merged as is to change using `errors.new()` to `wskderrors`, we can take care of it.